lib/jwt/jwt.go

```diff
@@ -218,6 +218,33 @@ func (k *Key) SignAzureToken(claims AzureTokenClaims) (string, error) {
 	return k.signAny(claims)
 }
 
+type PROXYSignParams struct {
+	ClusterName        string
+	SourceAddress      string
+	DestinationAddress string
+}
+
+const expirationPROXY = time.Second * 60
+
+func getPROXYSubjectClaim(sourceAddress, destinationAddress string) string {
+	return fmt.Sprintf("%s/%s", sourceAddress, destinationAddress)
```

Comment:
Are we 100% sure there won't be any `/` in the addresses? And should we add some prefix to indicate that this is intended to be a PROXY source+dest claim?

Reply:
Also, can't we have source addr and dest addr in separate (custom) claims, and maybe have a static string for the Subject instead?

Reply:
> 

I did make a suggestion that the `sub` and `aud` fields could actually be ideal for this.

Reply:
I've changed it 👍 , addresses have their own fields now - source address goes to `sub` and destination address to `aud`